config/initializers/search.rb

```diff
@@ -0,0 +1,26 @@
+Rails.application.configure do
+  config_dir = Rails.root.join("config/search")
+
+  synonyms = YAML.load_file(config_dir.join("synonyms.yml"))
+  oneway_synonyms = YAML.load_file(config_dir.join("oneway_synonyms.yml"))
```

Comment:
I've struggled to come up with a different name for `terms_with_synonyms` and `terms_with_oneway_synonyms`. Maybe we could rename `synonyms` and `oneway_synonyms` instead to `thesaurus` and `one_way_thesaurus`, since the difference is that these contain information about relationships and mappings, whereas the `terms_with*` are lists of words that can be looked up in the thesauruses.

Reply:
Ditto. I think "thesaurus" is nice, but on the other hand "synonym" is a bit more approachable.

I will sleep on it!

Reply:
Found some slightly better names – we can always improve them further later 😄